pkgs/desktops/gnome-3/extensions/taskwhisperer/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, taskwarrior }:
+
+stdenv.mkDerivation rec {
+  name = "gnome-shell-extension-taskwhisperer-${version}";
+  version = "11";
+
+  src = fetchFromGitHub {
+    owner = "cinatic";
+    repo = "taskwhisperer";
+    rev = "v${version}";
+    sha256 = "1g1301rwnfg5jci78bjpmgxrn78ra80m1zp2inhfsm8jssr1i426";
+  };
+
+  propagatedBuildInputs = [ taskwarrior ];
+
+  uuid = "taskwhisperer-extension@infinicode.de";
+
+  installPhase = ''
+    sed -i "s,\['task',\['${taskwarrior}/bin/task',g" taskService.js
```

Comment:
Also it would be better to use patch and `substituteAll` – it makes it easier to track if the replacement is necessary. See for example https://github.com/NixOS/nixpkgs/blob/f2fd4a5219fc4fc0d10cd29a842f0c1683095bdb/pkgs/applications/video/gnome-mplayer/default.nix#L18-L24